spectacles/runner.py

```diff
@@ -31,13 +32,43 @@ def __init__(
         port: int = 19999,
         api_version: float = 3.1,
         remote_reset: bool = False,
+        manifest_dependency: bool = False,
     ):
         self.project = project
+        self.manifest_dependency = manifest_dependency
         self.client = LookerClient(
             base_url, client_id, client_secret, port, api_version
         )
         self.client.update_session(project, branch, remote_reset)
 
+    def manage_dependent_branches(fn: Callable) -> Callable:  # type: ignore
```

Comment:
I've run into two things here: 

The first is `TypeError: 'staticmethod' object is not callable`.

If I then remove `@staticmethod`, I then get `NameError: name 'Runner' is not defined` because we are calling it before it's created as a class.

Reply:
If I move the function outside the class, it works. Is there a problem with that? It feels odd because it is using information about the class but isn't defined within it.

Reply:
Oh interesting... so that example doesn't actually run then. I agree that it's a bit odd but I think it's reasonably clear what it's supposed to do when you're reading it. I think you can leave it outside the class.